fili-core/src/main/java/com/yahoo/bard/webservice/druid/client/impl/AsyncDruidWebServiceImpl.java

```diff
@@ -61,17 +64,38 @@
     public static final String DRUID_WEIGHTED_QUERY_TIMER = DRUID_TIMER + "_W_";
     public static final String DRUID_SEGMENT_METADATA_TIMER = DRUID_TIMER + "_S_0";
 
-
+    private final Supplier<Map<String, String>> anyHeaders;
     private final DruidServiceConfig serviceConfig;
```

Comment:
Since we're adding the headers (and it looks like added headers strictly add, don't replace), I think a name like `headersToAdd` or `headersToAppend` helps clarify this.